modules/jdbc/build.gradle

```diff
@@ -9,7 +9,9 @@ dependencies {
     testImplementation 'org.vibur:vibur-dbcp:25.0'
     testImplementation 'org.apache.tomcat:tomcat-jdbc:10.0.23'
     testImplementation 'com.zaxxer:HikariCP-java6:2.3.13'
-    testImplementation 'com.googlecode.junit-toolbox:junit-toolbox:2.4'
     testImplementation 'org.rnorth.visible-assertions:visible-assertions:2.1.2'
     testImplementation 'org.assertj:assertj-core:3.23.1'
+    testImplementation ('org.mockito:mockito-core:4.6.1') {
+        exclude(module: 'hamcrest-core')
+    }
```

Comment:
Why did you add Mockito here to this module and exclude hamcrest from it?

Reply:
No special reason from me. I just copied from `core/build.gradle` and `modules/junit-jupiter/build.gradle`

Did not investigate any further why hamcrest-core is excluded from those modules.

Reply:
junit-toolbox brings mockito and it's being used in the jdbc module. so adding mockito make sense and since we are not using hamcrest-core it is ok to exclude it.